operator/controller/src/test/java/io/apicurio/registry/operator/it/SmokeITTest.java

```diff
@@ -40,5 +53,99 @@ void demoDeployment() {
                     .getSpec().getClusterIP()).isNotBlank();
             return true;
         });
+
+        // Ingresses
+        await().ignoreExceptions().until(() -> {
+            assertThat(client.network().v1().ingresses().inNamespace(getNamespace())
+                    .withName("demo-app-ingress").get().getSpec().getRules().get(0).getHost())
+                    .isEqualTo(registry.getSpec().getAppHost());
+            assertThat(client.network().v1().ingresses().inNamespace(getNamespace())
+                    .withName("demo-ui-ingress").get().getSpec().getRules().get(0).getHost())
+                    .isEqualTo(registry.getSpec().getUiHost());
+            return true;
+        });
+    }
+
+    @Test
+    void testService() {
+        // spotless:off
+        var registry = new ApicurioRegistry3();
+        var meta = new ObjectMeta();
+        meta.setName("demo");
+        meta.setNamespace(namespace);
+        registry.setMetadata(meta);
+        registry.setSpec(ApicurioRegistry3Spec.builder()
+                .appHost(ingressManager.getIngressHost("demo-app"))
+                .uiHost(ingressManager.getIngressHost("demo-ui"))
+                .build());
+        // spotless:on
+
+        // Act
+        client.resources(ApicurioRegistry3.class).inNamespace(namespace).create(registry);
+
+        // Wait for Services
+        await().ignoreExceptions().until(() -> {
+            assertThat(client.services().inNamespace(namespace).withName("demo-app-service").get().getSpec()
+                    .getClusterIP()).isNotBlank();
+            assertThat(client.services().inNamespace(namespace).withName("demo-ui-service").get().getSpec()
+                    .getClusterIP()).isNotBlank();
+            return true;
+        });
+
+        int appServicePort = portForwardManager.startPortForward("demo-app-service", 8080);
+
+        await().ignoreExceptions().until(() -> {
+            given().get(new URI("http://localhost:" + appServicePort + "/apis/registry/v3/system/info"))
+                    .then().statusCode(200);
+            return true;
+        });
+
+        int uiServicePort = portForwardManager.startPortForward("demo-ui-service", 8080);
+
+        await().ignoreExceptions().until(() -> {
+            given().get(new URI("http://localhost:" + uiServicePort + "/config.js")).then().statusCode(200);
+            return true;
+        });
+    }
+
+    @Test
+    void testIngress() {
+        if (ingressManager.isIngressSupported()) {
```

Comment:
nitpick: an idiomatic way of doing this would be to use something like: `@EnabledIfSystemProperty` or similar from JUnit